charts/alfresco-search-enterprise/templates/_helpers-elasticsearch.tpl

```diff
@@ -1,14 +1,14 @@
 {{- define "alfresco-search-enterprise.searchIndexExistingSecretName" -}}
-{{ coalesce .Values.searchIndex.existingSecretName .Values.global.elasticsearch.existingSecretName (printf "%s-elasticsearch-secret" (include "alfresco-search-enterprise.fullname" .)) }}
+{{- coalesce .Values.searchIndex.existingSecretName .Values.global.elasticsearch.existingSecretName (printf "%s-elasticsearch-secret" (include "alfresco-search-enterprise.fullname" .)) }}
 {{- end -}}
 
 {{- define "alfresco-search-enterprise.config.spring" -}}
-{{- if .Values.elasticsearch.enabled }}
+{{- if .Values.elasticsearch.enabled -}}
```

Comment:
is the closing `-` really necessary?

Reply:
that's cosmetic tbh 💅 ut I would argue that a named template should never start with a carriage return.